### PR TITLE
Update GameJoinS2CPacket.mapping

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: mappings/net/minecraft/network/packet/s2c/play/GameJoinS2CPacket.mapping [hunk 1]
```diff
@@ -5,17 +5,25 @@ CLASS net/minecraft/class_2678 net/minecraft/network/packet/s2c/play/GameJoinS2C
...
 	FIELD field_12282 gameMode Lnet/minecraft/class_1934;
 	FIELD field_12284 registryManager Lnet/minecraft/class_5455$class_5457;
-	FIELD field_19145 chunkLoadDistance I
+	FIELD field_19145 viewDistance I
 	FIELD field_20665 sha256Seed J
 	FIELD field_20666 flatWorld Z
 	FIELD field_24618 reducedDebugInfo Z
 	FIELD field_24619 showDeathScreen Z
 	FIELD field_25134 dimensionId Lnet/minecraft/class_5321;
+	FIELD field_25320 dimensionsIds Ljava/util/Set;
```
Comment (liach) [inline mappings/net/minecraft/network/packet/s2c/play/GameJoinS2CPacket.mapping]:
```suggestion
	FIELD field_25320 dimensionIds Ljava/util/Set;
```

File: mappings/net/minecraft/network/packet/s2c/play/GameJoinS2CPacket.mapping [hunk 2]
```diff
@@ -27,9 +35,12 @@ CLASS net/minecraft/class_2678 net/minecraft/network/packet/s2c/play/GameJoinS2C
...
 	METHOD method_11565 getRegistryManager ()Lnet/minecraft/class_5455;
 	METHOD method_11568 isFlatWorld ()Z
-	METHOD method_20204 getChunkLoadDistance ()I
+	METHOD method_20204 getViewDistance ()I
 	METHOD method_22423 getSha256Seed ()J
 	METHOD method_22424 showsDeathScreen ()Z
 	METHOD method_28118 isHardcore ()Z
 	METHOD method_28119 isDebugWorld ()Z
 	METHOD method_29176 getDimensionId ()Lnet/minecraft/class_5321;
+	METHOD method_29443 getDimensionsIds ()Ljava/util/Set;
```
Comment (liach) [inline mappings/net/minecraft/network/packet/s2c/play/GameJoinS2CPacket.mapping]:
```suggestion
	METHOD method_29443 getDimensionIds ()Ljava/util/Set;
```

**Comment by Leo40Git** [comment]
Link to another `dimensionsIds` that should be `dimensionIds`: https://github.com/FabricMC/yarn/pull/1693/files#diff-6a22d519560ac2eb75d7c5b07339f3c6R23

File: mappings/net/minecraft/network/packet/s2c/play/GameJoinS2CPacket.mapping [hunk 3]
```diff
@@ -5,17 +5,25 @@ CLASS net/minecraft/class_2678 net/minecraft/network/packet/s2c/play/GameJoinS2C
...
+	FIELD field_25320 dimensionsIds Ljava/util/Set;
+	FIELD field_25321 dimensionType Lnet/minecraft/class_2874;
+	FIELD field_25713 previousGameMode Lnet/minecraft/class_1934;
 	METHOD <init> (ILnet/minecraft/class_1934;Lnet/minecraft/class_1934;JZLjava/util/Set;Lnet/minecraft/class_5455$class_5457;Lnet/minecraft/class_2874;Lnet/minecraft/class_5321;IIZZZZ)V
 		ARG 1 playerEntityId
 		ARG 2 gameMode
+		ARG 3 previousGameMode
 		ARG 4 sha256Seed
 		ARG 6 hardcore
+		ARG 7 dimensionsIds
```
Comment (liach) [inline mappings/net/minecraft/network/packet/s2c/play/GameJoinS2CPacket.mapping]:
```suggestion
		ARG 7 dimensionIds
```

**Review by liach — APPROVED** [review]

**Review by YanisBft — APPROVED** [review]

**Pull request merged**